desktop/window-handlers/get-logs/index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const { app, dialog } = require( 'electron' );
+
+/**
+ * Internal dependencies
+ */
+const state = require( 'lib/state' );
+const system = require( 'lib/system' );
+const { zipContents } = require( 'lib/archiver' );
+const log = require( 'lib/logger' )( 'desktop:get-logs' );
+
+/**
+ * Module variables
+ */
+const logPath = state.getLogPath();
+
+function pad( n ) {
+	return n < 10 ? '0' + n : n;
+}
+
+const localDateTime = () => {
+	const now = new Date();
+	return now.getFullYear() +
+		'-' +
+		pad( now.getMonth() + 1 ) +
+		'-' +
+		pad( now.getDate() ) +
+		'T' +
+		pad( now.getHours() ) +
+		'.' +
+		pad( now.getMinutes() ) +
+		'.' +
+		pad( now.getSeconds() ) +
+		'.' +
+		pad( now.getMilliseconds() );
+}
+
+module.exports = async function( window ) {
+	const onZipped = async ( file ) => {
+		await dialog.showMessageBox( window, {
+			type: 'info',
+			buttons: [ 'OK' ],
+			title: 'Logs saved to your desktop',
+			message: 'Logs saved to your desktop' +
+				'\n\n' +
+				`${ path.basename( file ) }`,
+			detail: 'For help with an issue, please contact help@wordpress.com and share your logs.'
+		} )
+	}
+
+	const onError = async ( error ) => {
+		await dialog.showErrorBox(
+			'Error zipping activity logs.' +
+			'\n\n' +
+			'Please contact help@wordpress.com and mention the error details below:' +
+			'\n\n' +
+			error.stack +
+			'\n\n' +
+			'System info: ' + JSON.stringify( system.getVersionData() )
+		)
+	}
+
+	try {
+		const timestamp = localDateTime();
```

Comment:
This also works, and provides the additional certainty that the dates are referenced against UTC and won't be confused by any local timezone setting or override in the browser.

```js
const timestamp = new Date().toISOString();
```

Reply:
The intent was to have a localized timestamp (non-UTC), but with similar formatting. I tried scouring for solutions on this, and short of importing something like `momjentjs` (which would be overkill here) the only way I could find was to generate each component in this manner. Would be great to do this in a more concise way, though.